### PR TITLE
Add option OFF_ALWAYS to setting nav_overrides_motor_stop

### PR DESCRIPTION
**Pull request by shellixyz**
Related issues: #3195 #4077

### CODE REVIEW

**Comment by digitalentity** [comment]
Isn't it cleaner to add a new value (i.e. `OFF_ALWAYS`) to `nav_overrides_motor_stop` setting?

**Comment by shellixyz** [comment]
This way you couldn't chose what it will do outside of a FS event

**Comment by shellixyz** [comment]
Hmm thinking about you are right it doesn't make much sense to have nav_overrides_motor_stop to something else than OFF with nav_overrides_motor_stop_on_fs to OFF too

**Comment by digitalentity** [comment]
> Hmm thinking about you are right it doesn't make much sense to have nav_overrides_motor_stop to something else than OFF with nav_overrides_motor_stop_on_fs to OFF too

Yes, that's my thinking as well. If you want to keep motor stopped in case of failsafe, you probably want it stopped in all other cases as well. We're losing some flexibility, but also making things a bit simpler to configure

**Comment by shellixyz** [comment]
I removed the new setting and instead added a new option OFF_ALWAYS for nav_overrides_motor_stop

File: src/main/fc/settings.yaml [hunk 1]
```diff
@@ -2107,7 +2107,7 @@ groups:
...
+        description: "When set OFF the navigation system will not take over the control of the motor if the throttle is low (motor will stop). When set to OFF always the navigation system will not take over the control of the motor if the throttle was low even when failsafe is triggered. When set to AUTO_ONLY the navigation system will only take over the control of the throttle in autonomous navigation modes (NAV WP and NAV RTH). When set to ALL_NAV (default) the navigation system will take over the control of the motor completely and never allow the motor to stop even when the throttle is low. This setting only has an effect combined with MOTOR_STOP and is likely to cause a stall if fw_min_throttle_down_pitch isn't set correctly or the pitch estimation is wrong for fixed wing models when not set to ALL_NAV"
```
Comment (stronnag) [inline src/main/fc/settings.yaml]: `OFF always` should be `OFF_ALWAYS`, otherwise it's a bit confusing.

Comment (shellixyz) [reply src/main/fc/settings.yaml]: Good catch. Fixed

**Comment by ABLomas** [comment]
According to INAV TG channel - final 2.6 will be released tomorrow, but:
 - nav_overrides_motor_stop is mentioned in release notes, but at least in RC3 it does not prevent motor to be used in RTH mode (tested)
 - probably proper fix is available in this PR, but it's not part of 2.6 release.
If i test it tomorrow - any chance that it could be merged with 2.6, or you will just remove nav_overrides_motor_stop mention from release notes and this PR will be merged later?

**Pull request merged**